atd-vzd/migrations/default/1712941514048_delete_no_data_option/up.sql

```diff
@@ -0,0 +1,4 @@
+-- Remove the custom "NO DATA" lookup option, instead we should just use null to mean null
+DELETE FROM public.atd_txdot__veh_body_styl_lkp WHERE veh_body_styl_id = -1;
+
+UPDATE public.atd_txdot_units SET veh_body_styl_id = null WHERE veh_body_styl_id = -1;
```

Comment:
lolz that awkward moment when we dont have foreign key constraints so i was able to do the first deletion and then there were still units that had -1 for veh_body_styl_id even though we dont have a lookup value for that anymore :woozy_face:

Reply:
also btw this second UPDATE command only effects 3 record units. there are only 3 units where the body style has an id of -1